src/app/frontend/deploy/deploy.html

```diff
@@ -15,26 +15,19 @@
 -->
 
 <kd-content-card>
-  <kd-title>[[Deploy a Containerized App|Title text which appears on top of the deploy page.]]</kd-title>
-  <kd-content class="kd-content-padded">
-    <kd-help-section>
-      <md-radio-group ng-model="ctrl.selection"
-                      ng-change="ctrl.changeSelection()">
-        <md-radio-button value="{{::ctrl.appOption}}"
-                         class="md-primary">
-          [[Specify app details below|Text for a selection option, which the user must click to manually enter the app details on the deploy page.]]
-        </md-radio-button>
-        <md-radio-button value="{{::ctrl.fileOption}}"
-                         class="md-primary">
-          [[Upload a YAML or JSON file|Text for a selection option, which the user must click to upload a YAML/JSON file to deploy from on the deploy page.]]
-        </md-radio-button>
-      </md-radio-group>
-      <kd-user-help>
-        [[To learn more, <a href="http://kubernetes.io/docs/user-guide/ui/"
-           target="_blank"
-           tabindex="-1"> take the Dashboard Tour <i class="material-icons">open_in_new</i></a>|User help with a link redirecting to the "Dashboard tour" on the deploy page.]]
-      </kd-user-help>
-    </kd-help-section>
-    <div ui-view></div>
+  <kd-content>
+    <md-tabs md-dynamic-height
+             md-border-bottom
+             class="kd-deploy-tabs">
+      <md-tab label="[[Upload from input|Upload from input tab on deploy page.]]">
+        <kd-deploy-from-input></kd-deploy-from-input>
+      </md-tab>
+      <md-tab label="[[Upload from file|Upload from file tab on deploy page.]]">
+        <kd-deploy-from-file></kd-deploy-from-file>
+      </md-tab>
+      <md-tab label="[[Specify deployment details|Specify deployment details tab on deploy page.]]">
```

Comment:
I'd consider some shorter and simpler title here i.e. 'Create deployment', 'Create new resource/deployment'

Reply:
You can create deployments from a file and from text input, so I think it is too generic to put it here.

Reply:
Material writing guidelines say that used language should be concise, simple and direct. In this case first two tabs are fine. They simply say `Upload` which means that we are creating a new resource and `from x` to say that new resource is created based on some provide data.

The last tabs description is a bit misleading and IMO does not use simple and direct language. It says to specify details but does not stress out that it is a creation of a new resource based on those "specified details".

I have provided only sample descriptions. We can think of something else here.

Reply:
`Specify deployment details` is more clear to me than `Create deployment`. I considered a change to `Specify app details` but I suppose that you will not like it. I am not sure what is not simple or concise in `Specify deployment details` phase, so I will let you propose something else.

> It says to specify details but does not stress out that it is a creation of a new resource based on those "specified details".

It is a tab on create page. I think users will be aware that this is resource creation. Besides that upload is not equal to create too.

Reply:
> It is a tab on create page. I think users will be aware that this is resource creation. Besides that upload is not equal to create too.

In this case `upload == create`. Usually, upload means that you upload file or some data somewhere where you can read it from later. In our case upload means create from file/yaml/json. After all, it is the same as `kubectl create -f xxx`. 

I'd rather try to be persistent here and try to use same template in all cases.
`Create from file`
`Create from input` or `Create from yaml/json`
`Create deployment`

There is also option to move creation info higher so on the form there would be title i.e. `Resource creator` and then show some options, `File`, `YAML/JSON`, `Deployment` or `App`(app is actually better because we are creating multiple resources, not only single Deployment).

Reply:
> There is also option to move creation info higher so on the form there would be title i.e. Resource creator and then show some options

Would need even more design work.

> I'd rather try to be persistent here and try to use same template in all cases.
>Create from file
>Create from input or Create from yaml/json
>Create deployment or Create an app

I have applied your suggestions, but I do not agree with them. I liked the previous version more. We can change it later.

Reply:
@rahuldhide @danielromlein could you share your thoughts on this?